### PR TITLE
Add GitHub actions workflow

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      run: |
+        go install .
+        go test -v test/vndr_test.go
+      working-directory: src/github.com/LK4D4/vndr
+  lint:
+    name: Linters
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        go-version: ["1.16.x"]
```
Comment (thaJeztah) [inline .github/workflows/test.yml:44]: Kept this at go1.16 for now; on go1.17 it gets confused by the `// +build` examples in the docs (will need to have a look if we can add support for go1.17-style `//go:build` tags 😞

**Comment by thaJeztah** [comment]
Test-run can be seen on https://github.com/thaJeztah/vndr/pull/1

**Comment by thaJeztah** [comment]
@LK4D4 PTAL

**Comment by LK4D4** [comment]
@thaJeztah will this work as is or do I need to setup something (maybe send me a doc or something)?
Can I retire travis after this?

**Comment by thaJeztah** [comment]
I don't think any setting changes will be needed (unless you want to make the check "required" in the branch protection settings).

After this is merged, if should automatically run on pull requests, so we can remove the travis configuration file after that (although I think travis was already non-functional because they deprecated travis.org)

**Pull request merged**